nucypher/policy/conditions/evm.py

```diff
@@ -281,9 +280,15 @@ def _configure_provider(self, *args, **kwargs):
 
     def _get_unbound_contract_function(self) -> ContractFunction:
         """Gets an unbound contract function to evaluate for this condition"""
+        function_abi = _resolve_abi(
```

Comment:
👍

Reply:
This is a sensible place to resolve an ABI since we're attaching a callable contract function. 👍🏻